docs/source/release_notes/release-notes.rst

```diff
@@ -66,42 +90,39 @@ develop
            Existing installations will be unaffected.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/3088>`__)
 
-    *    - |fixed| |improved|
-         - AtlasDB now partitions versions of cells to be swept into batches more robustly and more efficiently.
-           Previously, this could cause stack overflows when sweeping a very wide row, because the partitioning algorithm attempted to traverse a recursive hierarchy of sublists.
-           Also, previously, partitioning would require time quadratic in the number of versions present in the row; it now takes linear time.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/3095>`__)
+    *    - |fixed|
+         - OkHttp is not handling ``Thread.interrupt()`` well, and calling interrupts repeatedly may cause corrupted clients.
+           To avoid this issue, our Feign client is now wrapped with an ``ExceptionCountingRefreshingClient``, which will detect and refresh corrupted clients.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/3121>`__)
```

Comment:
For this one don't know if it's worth emphasising that the corruption is only in the http client (i.e. if I was running a previous Atlas version and interrupted things, it shouldn't have compromised data integrity). At least as far as I know the damage was limited to that

Reply:
Makes sense - the current wording could be a bit scary. I'll amend + merge.